There are some problems on CI related to python executable, but it is only a warning. I don't know if this problem is caused by the new version or the problem existed before.

```
-- The C compiler identification is GNU 11.4.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Performing Test Iconv_IS_BUILT_IN
-- Performing Test Iconv_IS_BUILT_IN - Success
-- Found Iconv: built in to C library
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found ZLIB: optimized;/mnt/vcpkg-ci/installed/x64-linux/lib/libz.a;debug;/mnt/vcpkg-ci/installed/x64-linux/debug/lib/libz.a (found version "1.3.1")
-- Looking for include file unistd.h
-- Looking for include file unistd.h - found
-- Looking for F_GETFL
-- Looking for F_GETFL - found
CMake Error at /vcpkg/downloads/tools/cmake-3.30.1-linux/cmake-3.30.1-linux-x86_64/share/cmake-3.30/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  Could NOT find Python (missing: Python_EXECUTABLE Python_INCLUDE_DIRS
  Python_LIBRARIES Interpreter Development Development.Module
  Development.Embed)
Call Stack (most recent call first):
  /vcpkg/downloads/tools/cmake-3.30.1-linux/cmake-3.30.1-linux-x86_64/share/cmake-3.30/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
  /vcpkg/downloads/tools/cmake-3.30.1-linux/cmake-3.30.1-linux-x86_64/share/cmake-3.30/Modules/FindPython.cmake:667 (find_package_handle_standard_args)
  /vcpkg/scripts/buildsystems/vcpkg.cmake:893 (_find_package)
  CMakeLists.txt:65 (find_package)


-- Configuring incomplete, errors occurred!
ninja: build stopped: subcommand failed.
```

https://artprodwus21.artifacts.visualstudio.com/A6adfb432-9816-4cb2-8190-03885865e0d9/c1ee48cb-0df2-4ab3-8384-b1df5a79fe53/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL3ZjcGtnL3Byb2plY3RJZC9jMWVlNDhjYi0wZGYyLTRhYjMtODM4NC1iMWRmNWE3OWZlNTMvYnVpbGRJZC8xMTU4MjUvYXJ0aWZhY3ROYW1lL2ZhaWx1cmUrbG9ncytmb3IreDY0LWxpbnV40/content?format=zip

--- a/ports/libxslt/portfile.cmake
+++ b/ports/libxslt/portfile.cmake
@@ -1,14 +1,8 @@
-vcpkg_download_distfile(FIX_PACKAGE_VERSION_PATCH
-    URLS https://github.com/GNOME/libxslt/commit/7504032097712714aafe309d54f2ad57e3364bac.diff?full_index=1
-    FILENAME Fix-package-version.patch
-    SHA512 972921decf374fe8a4cad4e09890ce0d5961ee05e3c52d117c09fe8bde1a4540ebe212e767f8a95d281945240f29a90fd15e37104f45d47440032737d41dc8d0
-)
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO GNOME/libxslt
     REF "v${VERSION}"
-    SHA512 0974419e0eae3cd4070ca52341b3df2d1b873b30d0ede2143274fcd0ef8653d5ac55b5f0faad56d8cf60443fefb01c5f5ddecff4b7638ba28e450e88f1c3d3c4
+    SHA512 51d9e9586f78c5aa69ac67fac64b865625fefb16bf06f1f06dede0a57b3e382e78dea69145c7c0c59f06735b738bed209751e691dd9045c3cc33df096963f89d
     HEAD_REF master
     PATCHES
         python3.patch
@@ -16,7 +10,6 @@ vcpkg_from_github(
         libexslt-pkgconfig.patch
         fix-gcrypt-deps.patch
         skip-install-docs.patch
-        ${FIX_PACKAGE_VERSION_PATCH}
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS

--- a/ports/libxslt/vcpkg.json
+++ b/ports/libxslt/vcpkg.json
@@ -1,7 +1,6 @@
 {
   "name": "libxslt",
-  "version": "1.1.42",
-  "port-version": 2,
+  "version": "1.1.43",
   "description": "Libxslt is a XSLT library implemented in C for XSLT 1.0 and most of EXSLT",
   "homepage": "https://github.com/GNOME/libxslt",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5649,8 +5649,8 @@
       "port-version": 0
     },
     "libxslt": {
-      "baseline": "1.1.42",
-      "port-version": 2
+      "baseline": "1.1.43",
+      "port-version": 0
     },
     "libxt": {
       "baseline": "1.3.0",

--- a/versions/l-/libxslt.json
+++ b/versions/l-/libxslt.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "eb1445468e9d858f0ff63cc777b48e98c4d10a72",
+      "version": "1.1.43",
+      "port-version": 0
+    },
+    {
       "git-tree": "00e5829fca614f1b82dc9fa5439ef0a8d35d7cb4",
       "version": "1.1.42",
       "port-version": 2

